tox-travis.ini

```diff
@@ -6,15 +6,10 @@
 # tox.ini and tox-travis.ini are copies of the same content with different envlist commented out
 # tox-travis.ini will have pyXX-clean and all pyXX-installers in the default envlist, while the developer tox.ini
 # does not have clean and only has one pyXX-installers
-# Historically we've used the latest Python for environments that we only list one Python for. For now, we're using Python 3.12
-# to avoid updating our pb2 files, which could possibly cause incompatibilities with other NI Python packages.
-# When other NI Python packages bump the version that they use for generating pb2 files, we can bump this Python version to match.
-# We may also bump it sooner, if we agree with team members who own other NI Python packages that it's okay to do so.
-# At the latest, we'll bump it when we drop support for Python 3.12.
 # Uncomment this line for tox.ini
-# envlist = py312-build_test,py312-codegen,py312-installers,py{39,310,311,312,313}-test,py312-flake8,py312-docs,py312-pkg
+# envlist = py312-build_test,py312-codegen,py312-installers,py{38,39,310,311,312}-test,py312-flake8,py312-docs,py312-pkg
```

Comment:
Nope. We're not restoring Python 3.8 support or dropping Python 3.12 support.

Reply:
You're overrwriting build_release.py, tox-travis.ini and tox.ini with old versions of the files.

Reply:
You haven't fixed this.
Copy tox-travis.ini, and tox.ini from main and overwrite what you have then reimplement your nirfsg changes.